nion/swift/ExportDialog.py

```diff
@@ -221,56 +223,163 @@ def cancel(self) -> bool:
         return True
 
 
-class ExportSVGHandler:
-    def __init__(self, display_size: Geometry.IntSize) -> None:
-        self.width_model = Model.PropertyModel(display_size.width)
-        self.height_model = Model.PropertyModel(display_size.height)
+class UnitType(enum.Enum):
+    INCHES = 0
+    CENTIMETERS = 1
+    PIXELS = 2
 
-        self.int_converter = Converter.IntegerToStringConverter()
+class ExportSVGHandler:
```

Comment:
There should be exactly two blank lines before each top-level class. Please add one here.